docs/docs/layout-animations/layout-animation-config.mdx

```diff
@@ -0,0 +1,88 @@
+---
+sidebar_position: 5
+---
+
+# Skipping Layout Animations
```

Comment:
```suggestion
---
sidebar_position: 5
title: LayoutAnimationConfig
sidebar_label: Skipping Layout Animations
---
```
Let's change it so in the sidebar says `Skipping Layout Animations` but the title of the page is `LayoutAnimationConfig`